### PR TITLE
Fix experiment tracking

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

Fixes some issues found in experiment tracking testing:

- step of TB send/receive
- export PATH

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

**Review by nvkevlu — APPROVED** [review]
Thank you for fixing these, these look good, and I can look into log_text.

File: examples/advanced/experiment-tracking/pt/learner_with_mlflow.py [hunk 1]
```diff
@@ -168,10 +168,11 @@ def local_train(self, fl_ctx, abort_signal):
                         fl_ctx, f"Epoch: {epoch}/{self.epochs}, Iteration: {i}, " f"Loss: {running_loss/3000}"
                     )
                     running_loss = 0.0
-                    self.writer.log_text(
-                        f"last running_loss reset at '{len(self.train_loader) * epoch + i}' step",
-                        "running_loss_reset.txt",
-                    )
+                    # TODO (Kevin): Fix the following:
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply examples/advanced/experiment-tracking/pt/learner_with_mlflow.py:171]: I added these comments, the old code does not work.

File: nvflare/app_common/widgets/streaming.py [hunk 2]
```diff
@@ -103,10 +103,11 @@ def handle_event(self, event_type: str, fl_ctx: FLContext):
 
     def add(self, tag: str, value, data_type: AnalyticsDataType, global_step: Optional[int] = None, **kwargs):
         kwargs = kwargs if kwargs else {}
-        global_step = kwargs.get("global_step", None)
-        if global_step:
+        if global_step is not None:
             if not isinstance(global_step, int):
                 raise TypeError(f"Expect global step to be an instance of int, but got {type(global_step)}")
+            kwargs[TrackConst.GLOBAL_STEP_KEY] = global_step
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/app_common/widgets/streaming.py:109]: case 1: 
global_step = None, kwargs = None

case 2:
global_step = 2, kwargs = None

case 3:
global_step = None, kwargs = {"global_step": 3}

case 4 (should not happen):
global_step = 10, kwargs = {"global_step": 3}


The old code is not working, @nvkevlu need to make sure new code correctly handle the above case.

I can close this PR, this PR just showcase the quick fix that I did.
It required to be polished by Kevin.

**Comment by YuanTingHsieh** [comment]
@nvkevlu i just close this PR.

please cherry pick the changes and improve on top of this, thanks!

**Pull request closed without merging**